docs/en/siem/machine-learning.asciidoc

```diff
@@ -66,404 +69,10 @@ Management -> Advanced Settings -> `siem:defaultAnomalyScore`.
-
-* Winlogbeat (Windows)
-
-+
-Required ECS fields:
-
-* host.name
-* process.name
-* process.parent.name
-* user.name
-* event.action
-* agent.type
-
-windows_anomalous_script:: Searches for PowerShell scripts with unusual data
-characteristics, such as obfuscation, that may be a characteristic of malicious
-PowerShell script text blocks.
-+
-Beats required on hosts:
-
-* Winlogbeat (Windows)
-
-NOTE: You can only run this job when Winlogbeat is used to ship data.
-
-windows_anomalous_service:: Searches for unusual Windows services that could 
-indicate execution of unauthorized services, malware, or persistence mechanisms.
-+
-In corporate Windows environments, hosts do not generally run many rare or 
-unique services. This job helps detect malware and persistence mechanisms that 
-have been installed and run as a service.
-+
-Beats required on hosts:
-
-* Winlogbeat (Windows)
-
-NOTE: You can only run this job when Winlogbeat is used to ship data.
-
-windows_anomalous_user_name_ecs::
-+
-linux_anomalous_user_name_ecs:: Searches for activity from users who are not 
-normally active, which could indicate unauthorized changes, activity by 
-unauthorized users, lateral movement, and compromised credentials.
-+
-In organizations, new usernames are not often created apart from specific types 
-of system activities, such as creating new accounts for new employees. These 
-user accounts quickly become active and routine.
-+
-Events from rarely used usernames can point to suspicious activity. 
-Additionally, automated Linux fleets tent to see activity from rarely used 
-usernames only when personnel log in to make authorized or unauthorized 
-changes, or threat actors have acquired credentials and log in for malicious 
-purposes. Unusual usernames can also indicate pivoting, where compromised 
-credentials are used to try and move laterally from one host to another.
-+
-Beats required on hosts:
-
-* Auditbeat (Linux)
-* Winlogbeat (Windows)
-
-+
-Required ECS fields:
-
-* host.name
-* process.name
-* user.name
-* event.action
-* agent.type
-
-windows_rare_user_runas_event:: Searches for unusual user context switches 
-using the `runas` command or similar techniques, which could indicate account 
-takeover or privilege escalation using compromised accounts. Privilege 
-elevation using tools like `runas` is more common for domain and network 
-administrators than professionals who are not members of the technology 
-department.
-+
-Beats required on hosts:
-
-* Winlogbeat (Windows)
-
-+
-Required ECS fields:
-
-* process.name
-* host.name
-* user.name
-* event.code
-* agent.type 
-
-windows_rare_user_type10_remote_login:: Searches for unusual remote desktop 
-protocol (RDP) logins, which could indicate account takeover or credentialed 
-persistence using compromised accounts. RDP attacks such as BlueKeep also tend 
-to use unusual usernames.
-+
-Beats required on hosts:
+hosts. To run jobs when you are not using {beats}, you must map your data to
```

Comment:
With other ML modules, you don't even see the job suggestions unless you have the appropriate data. If that's true here as well, it might help to do something like this:
```suggestion
hosts. If you are not using {beats}, you must map your data to
```

Reply:
In SIEM, you can always see the jobs. They are marked as unavailable. Changing to:
To enable jobs when you are not using {beats},...